### ISSUE TITLE
mpc tab breaks when folder contains spaces

### ISSUE THREAD

**Issue by Neo-Oli** [MusicPlayerDaemon/mpc#7]
In zsh, if I type:
```
mpc add Electronic/Las<tab>
mpc add Electronic/Laserkraft\ 3D/<tab>
mpc add Electronic/Laserkraft\ 3D/mpd error: No such directory
```
It seems to have a problem with spaces.  What I see is that `mpc tab "Electronic/Laserkraft 3D/"` and `mpc tab Electronic/Laserkraft\ 3D/` work, but `mpc tab "Electronic/Laserkraft\ 3D/"` leads to that error. Perhabs mpc tab could be made to treat `\<space>` as regular space. Otherwise perhaps https://github.com/zsh-users/zsh/blob/master/Completion/Unix/Command/_mpc needs to be changed.

I am running mpc 0.28 and zsh 5.4.1.

**Comment by MaxKellermann** [comment]
The directory "`Electronic/Laserkraft\ 3D/`" does not exist, does it? Looks like the error message is perfectly correct.
And if mpc would treat "`\<space>`" as regular space, then how would one be able to specify a backslash?

**Comment by Neo-Oli** [comment]
@MaxKellermann No the directory `Electronic/Laserkraft\ 3D/` does exist.
```
$ mpc tab "Electronic/Laserkraft 3D/"
Electronic/Laserkraft 3D/00 - Nein Mann.opus
$ mpc add Electronic/Laserkraft\ 3D/mpd error: No such directory

$ mpc add Electronic/Laserkraft\ 3D/00\ -\ Nein\ Mann.opus  #Writing it manually
$ mpc play
Laserkraft 3D - Nein Mann
[playing] #1/2   0:00/3:51 (0%)
volume:100%   repeat: on    random: off   single: off   consume: off
$
```

**Comment by MaxKellermann** [comment]
Your listing shows that "`Laserkraft 3D`" exists, but I don't see a directory called "`Laserkraft\ 3D`" (with a backslash in the name).

**Comment by Neo-Oli** [comment]
But the `\` gets added by zsh automatically, to escape the space. I made a video that demonstrates my problem: 
https://asciinema.org/a/vkNebrU6v5b9XYrMrdVeKpfB8

**Comment by MaxKellermann** [comment]
I'm not interested in watching your video, and I don't care who adds the backslash. What matters for me is: the backslash is wrong, and there's no bug in mpc. Now let's stop discussing this. Write a bug report when you find a bug in mpc, but please don't bother me with zsh bugs.

**Comment by Neo-Oli** [comment]
For future reference: This has been fixed in version 5.4.2 of zsh. [This](https://sourceforge.net/p/zsh/code/ci/93ce12e7a00208f2552a9b06248a4839b082ad56/) is the commit that fixes it.

**Comment by kaitlynia** [comment]
Not sure why the fallout here was so hostile. It's common to escape backslashes with a backslash `\\` and surprised this was not mentioned as a compromise.

EDIT: This also affects #140. I may open an issue to support escape sequences.
